src/components/accession2/edit/QuantityModal.tsx

```diff
@@ -172,7 +172,9 @@ export default function QuantityModal(props: QuantityModalProps): JSX.Element {
               onChange={(value) => onChangeRemainingQuantity('seedsQuantity', Number(value))}
               type='text'
               value={
-                record.remainingQuantity?.units === 'Seeds' ? record.remainingQuantity?.quantity : record.estimatedCount
+                record.remainingQuantity?.units === 'Seeds'
+                  ? record.remainingQuantity?.quantity.toString()
+                  : record.estimatedCount
```

Comment:
why toString for quantity but not estimatedCount? they are both 'numbers'

Reply:
True, I was thinking in the input value (that is remaining quantity), but estimated value can also be 0

Reply:
Oops, I meant why the toString conversion? Can we use them as numbers?

Reply:
The problem is that the the Textfield component has an optional "value", and 0 as number is consider false, so it doesn't render. That's why I'm converting the 0 to "0", so that the textfield shows the value

Reply:
Yes, I remember - in web-components.. we should probably fix that at some point for input type='number'?
And maybe use number as type in this text field.